src/types/orders.ts

```diff
@@ -47,6 +47,13 @@ export enum WorkflowTemplateID {
   SampleProject = 'Sample-project',
 }
 
+export enum TypesWithStartTime {
+  ORAL_TRANSLATION = 'ORAL_TRANSLATION',
```

Comment:
**[blocker]**

Would be good to use PascalCase for these enum keys for consistency

Reply:
fixed